packages/expo-auth-session/src/AuthSession.types.ts

```diff
@@ -37,7 +41,7 @@ export type AuthSessionResult =
       /**
        * How the auth completed.
        */
-      type: 'cancel' | 'dismiss' | 'locked';
+      type: 'cancel' | 'dismiss' | 'opened' | 'locked';
```

Comment:
@wschurman Any reason why `opened` was added? It seems like `type` would never be this value:
https://github.com/expo/expo/blob/5b22b6db863ac16c076375da011623adbbeeae52/packages/expo-auth-session/src/AuthRequest.ts#L192-L195

Reply:
It was added to fix a TypeScript error (can see it by running `yarn tsc` with this change reverted. While it may never happen at runtime, it's tough to express that at typecheck time.

Reply:
The issue is that it makes it seems like this is a valid result type and it appears undocumented. In my codebase I'm trying to handle all the possible return types, and this one is undocumented. I can help with the typing.